backend/src/main/java/com/stackunderflow/backend/security/SecurityConfig.java

```diff
@@ -26,7 +26,7 @@ public class SecurityConfig {
 
     private static final String[] ONLY_ACCESS_FOR_ADMIN = {""};
     private static final String[] ONLY_ACCESS_FOR_USER = {"/api/topic/all"};
-    private static final String[] UNSECURED_URLS = {"/api/badge/**", "/api/auth/**"};
+    private static final String[] UNSECURED_URLS = {"/api/badge/**", "/api/auth/**", "/api/post/**"};
```

Comment:
/api/post/** also includes the create post endpoint which might throw an exception for non auth users

Reply:
aici trebuie sa vedem la final care sunt cu auth si care nu